wire-ios-sync-engine/Tests/Source/UserSession/ZMUserSessionTests.swift

```diff
@@ -304,11 +304,7 @@ final class ZMUserSessionTests: ZMUserSessionTestsBase {
             results: [.success(())],
             context: syncMOC.notificationContext
         )
-        let pushSupportedProtocolsActionHandler = MockActionHandler<PushSupportedProtocolsAction>(
```

Comment:
this does not happen anymore because the fix will set supported protocols so during the resolve one on one conv, there is no protocols change therefore no push of protocols